src/hwrenderer/postprocessing/hw_postprocess_cvars.cpp

```diff
@@ -100,5 +100,15 @@ CUSTOM_CVAR(Bool, gl_paltonemap_reverselookup, true, CVAR_ARCHIVE | CVAR_NOINITC
 	screen->UpdatePalette();
 }
 
+CUSTOM_CVAR(Int, gl_paltonemap_colormap, 1, CVAR_ARCHIVE | CVAR_NOINITCALL)
+{
+    screen->UpdatePalette();
```

Comment:
Always do if (screen != nullptr) checks in these CVARs - they can be changed at the command prompt without the screen being active and this will cause a crash.